include/wx/math.h

```diff
@@ -181,4 +181,9 @@ inline double wxRadToDeg(double rad) { return (rad * 180.0) / M_PI; }
 /* Compute the greatest common divisor of two positive integers */
 WXDLLIMPEXP_BASE unsigned int wxGCD(unsigned int u, unsigned int v);
 
+#ifdef __cplusplus
+/* Count trailing zeros */
+WXDLLIMPEXP_BASE unsigned int wxCTZ(unsigned x);
```

Comment:
Could you please add documentation for this function (with the inevitable `@since 3.1.2`) to `interface/wx/math.h` please?

Reply:
Done.